front/public/locales/en/operationalStudies/manageTrainSchedule.json

```diff
@@ -34,6 +34,7 @@
     "noTrainStepInput": "Must be > 0",
     "softErrorInfra": "The application is unavailable, a server error has occurred, please try again later.",
     "The requested train couldn't reach its destination": "The train cannot reach its destination",
+    "trainScheduleNameInvalid": "Required field. The name must not exceed 128 characters",
```

Comment:
Do you think you can add 2 messages ? One 'Required field' and another one 'The name must not exceed 128 characters' ?

Reply:
Sure, it's done